### DISCUSSION TITLE
Welcome to Growthcraft-1.15 Discussions!

### DISCUSSION THREAD

**Discussion by Alatyami** [GrowthcraftCE/Growthcraft-1.15#80] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Tetrapod1206** [comment]
hello, I just found this mod from Github search, wondering if I can make some effort to the development.
I have 9 years experience playing minecraft(7 years with mod).

**Reply by Alatyami** [reply]
Always open to getting assistance. How would you like to contribute?

**Reply by Tetrapod1206** [reply]
Hmm... I have no clear idea lol. maybe I can begin from fixing some bugs that are already listed in issue?
not really sure if I have enough understanding to do it

**Reply by Alatyami** [reply]
Sure thing. Fork the development branch as it is the latest. The release branch is used to when we push out to Curse Forge. Let me know which Bug you pick up and if you have any questions. We're also on Discord and have a dev-chat channel. 

https://discord.gg/Quh76Jn
